frontend/src/shared/components/register-page/register-page.component.html

```diff
@@ -0,0 +1,85 @@
+                    </div>
+                    <div class="input-container">
+                        <div class="label-container full-width">
+                            <label>Password</label>
+                        </div>                        
+                        <div class="password-container">
+                            <input class="input full-width" [ngClass]="{
+                                'error': (passwordControl.invalid && (passwordControl.dirty || passwordControl.touched))
+                                }"
+                                [type]="hidePass ? 'password' : 'text'" 
+                                placeholder="Write password"
+                                [value]="password"
+                                [(ngModel)]="password"
+                                (ngModelChange)="resetPasswordControl()"
+                                formControlName="passwordControl"
+                              />
+                              <mat-icon class="hide-icon" (click)="hidePass = !hidePass">{{hidePass ? 'visibility' : 'visibility_off'}}</mat-icon>
+                        </div>                        
+                        <span class="error" *ngIf="passwordControl.errors?.['required'] && (passwordControl.dirty || passwordControl.touched)">Password is required.</span>
+                        <span class="error" *ngIf="passwordControl.errors?.['minlength']">Must be at least 8 characters</span>
+                    </div>
+                    <div class="input-container">
+                        <div class="label-container full-width">
+                            <label>Repeat password</label>
+                        </div>                        
+                        <div class="password-repeat-container">
+                            <input class="input full-width" [ngClass]="{
+                                'error': (passwordRepeatControl.invalid && (passwordRepeatControl.dirty || passwordRepeatControl.touched))
+                                }"
+                                [(ngModel)]="passwordRepeat"
+                                [type]="hidePassRepeat ? 'password' : 'text'" 
+                                placeholder="Repeat password"
+                                formControlName="passwordRepeatControl"
+                              />
+                              <mat-icon class="hide-icon" (click)="hidePassRepeat = !hidePassRepeat">{{hidePassRepeat ? 'visibilityRepeat' : 'visibility_off'}}</mat-icon>
+                        </div>
+                        <span class="error" *ngIf="passwordRepeatControl.errors?.['pattern'] && (passwordRepeatControl.dirty || passwordRepeatControl.touched)">Password do not match.</span>
+                    </div>
+                </form>
+            </div>
+            <div class="button-container">
+                <tasque-button text="Sign up" class="btn fill full-width" (btnClick)="submitForm()"></tasque-button>
```

Comment:
You should disable button if form is not valid

Reply:
Where you found that?

Reply:
Common sense? If data is invalid we should not bother sending it to backend only to receive 400 Bad Req

Reply:
And we can't. If form is invalid it will not send it to backend, but we need pop-up component to inform user about it